server/api/contests.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * Fetch from firebase all contests date, id, name, and year_end
```

Comment:
These are fetched from the database. I'm not using firebase for DB, only cloud storage.